### DISCUSSION TITLE
net::ERR_NAME_NOT_RESOLVED

### DISCUSSION THREAD

**Discussion by PhHitachi** [socketio/socket.io#4170] [Q&A] [answered]
Hi i have a problem the socket connecting wrong directory

somting like  `https://http/socket.io/?EIO=3&transport=polling&t=Nqa3_Uw` 

![image](https://user-images.githubusercontent.com/44565442/141801995-0372e9a8-62c4-48d3-a76e-9b8292c44803.png)

**Comment by PhHitachi** [comment, marked as answer]
ohh i figure it out

**Reply by somesh1234567** [reply]
what is the solution?
